opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -483,13 +483,15 @@ namespace Opm
                       const WellMap&    wmap)
         {
             std::vector<int> resv_prod;
-
-            for (int w = 0, nw = wells.number_of_wells; w < nw; ++w) {
-                if (is_resv_prod(wells, w) ||
-                    ((wells.name[w] != 0) &&
-                     is_resv_prod(wmap, wells.name[w], step)))
-                {
-                    resv_prod.push_back(w);
+            if( &wells )
```

Comment:
This must be wrong I think. The pointer to a reference must be nonzero, so this is a noop test. I think you must either pass a pointer to this function or place its call inside an `if` statement.